fourkitchens/gatsby-starter-emulsify-drupal
# 1) Image Minification and SVG spriting

Adds image minification, which minifies all files in the `images` directory (and subdirectories) and resaves them in place. Once we have a good workflow, we should enable this only on build to save dev time.

Also adds SVG spriting, which runs after image optimization and creates the sprite file in `/dist/icons.svg`.

**To test:**
- Add some images into the `images` directory and run `yarn build` and ensure the images are optimized.
- Add SVG icons into the `images/icons` directory and verify they are showing up in [Storybook](http://localhost:6006/?path=/story/atoms-icons--icons) when running `yarn develop`
